app/components/disclosure-triangle.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
if you are invoking this component with angle brackets, can't some of this be moved to the template instead?

Reply:
We cannot currently use angle bracket components in inspector. It will not build.